client/src/pages/search/SearchCustom.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from "react";
+import { useSelector  } from "react-redux";
+import searchStyles from "./Search.module.css";
+import { FileSelectComponent } from "../../components/fileSelect";
+import axios from "axios";
+
+
+const baseURL = "http://localhost:3000/";
```

Comment:
Same thing -- seems like we expect a lot of default ports -- maybe we can add a list/table to README